File: 0-0-intro/src/main/java/com/bobocode/intro/FindMax.java
package com.bobocode.intro;

public class FindMax {
    public static void main(String[] args) {
        System.out.println(findMax(new int[]{1, 2, 23, 24, 98, 4}));
        System.out.println(findMax(new int[]{}));
    }

    public static int findMax(int[] input) {
        if (0 == input.length) {
            throw new IllegalArgumentException("Array is empty");
        }
        int max = input[0];
        for (int i = 1; i < input.length; i++) {
            if (input[i] > max) {
                max = input[i];
            }
        }
        return max;
    }
}

File: 0-0-intro/src/main/java/com/bobocode/intro/findMax.js
function findMax(array) {
    let max;
    if (array) {
        max = array[0];
        for (let i = 0; i < array.length; i++) {
            if (array[i] > max) {
                max = array[i];
            }
        }
    }
    return max;
}

console.log(findMax([1, 2, 34, 23, 22]));
